cmd/agent/main_windows.go

```diff
@@ -54,15 +55,17 @@ func main() {
 		runService(false)
 		return
 	}
-	defer log.Flush()
+
+
+	// go_expvar server
+	go http.ListenAndServe("127.0.0.1:5000", http.DefaultServeMux)
```

Comment:
Should this be configurable? Like https://github.com/DataDog/datadog-agent/blob/ce4f25875196f526dadc9d25028ba7fbdd543f6f/cmd/agent/app/start.go#L146-L147 ?

Reply:
Shouldn't be there at all.  Was moved to `StartAgent()`.  Removed